trendico/views.py

```diff
@@ -123,3 +121,55 @@ def post(self, request):
     def get(self, request):
         form = SignUpForm()
         return render(request, self.template_name, {'form': form})
+
+
+class AddToCartView(LoginRequiredMixin, View):
```

Comment:
rather than using `cart_summary`
it can be part of this class e.g `CartView` with get() and post() func